FileIO/XmlIO/Boost/BoostXmlGmlInterface.cpp

```diff
@@ -33,6 +25,38 @@
 #include "GeoLib/Surface.h"
 #include "GeoLib/Triangle.h"
 
+
+namespace
+{
+
+//! Method for handling conversion to string uniformly
```

Comment:
I'd add "... to string uniformly across all types and std::string; see std::string overload below."